projects/BFDR/NatalityRecord_submissionProperties.cs

```diff
@@ -740,7 +740,12 @@ public Dictionary<string, string> PlaceOfBirth
         [Property("Mother's Place Of Birth", Property.Types.Dictionary, "Mother Demographics", "Mother's Place Of Birth.", true, VR.IGURL.Mother, true, 305)]
         [PropertyParam("addressState", "address, state")]
         [PropertyParam("addressCountry", "address, country")]
-        [FHIRPath("Bundle.entry.resource.where($this is Patient).extension.where(url='" + OtherExtensionURL.PatientBirthPlace + "')", "")]
+        // NOTE TODO: In order to distinguish between multiple patient records we would need to follow the subject
+        // reference; since following references is not available as part of the FHIR Path implementation in version
+        // 4.3.0 of the Hl7.Fhir.R4 library we fall back to using the Meta Profile URL, which is not guaranteed to
+        // be present but will either provide a correct result or blank; we can revisit this when the version of the
+        // FHIR library is updated to a version that supports following references
```

Comment:
Might need to update the note here- 
did you mention that the Hl7.Fhir.R4 library update wouldn't actually address this? Is the only option going to be an IG modification that adds specific roles like we added for certifier?

Reply:
My understanding is that the library update does address this (though I haven't tested it).